spec/acceptance/iis_site_spec.rb

```diff
@@ -69,7 +92,24 @@
           puppet_resource_should_show('ensure',               'started')
           puppet_resource_should_show('applicationpool',      'DefaultAppPool')
           puppet_resource_should_show('enabledprotocols',     'https')
-          puppet_resource_should_show('logflags',             ['Date', 'Time', 'ClientIP', 'UserName'])
+          #puppet_resource_should_show('bindings',             [
+          #    {
+          #      'bindinginformation'   => '*:8080:',
+          #      'certificatehash'      => '',
+          #      'certificatestorename' => '',
+          #      'protocol'             => 'http',
+          #      'sslflags'             => '0',
+          #    },
+          #    {
+          #      'bindinginformation'   => '*:8084:domain.test',
+          #      'certificatehash'      => '',
+          #      'certificatestorename' => '',
+          #      'protocol'             => 'http',
+          #      'sslflags'             => '0',
+          #    }
+          #  ]
+          #)
```

Comment:
Can't do this unless `puppet_resource_should_show` is made smarter for arrays of hashes... and using regexes to do so is icky.